csm_web/scheduler/models.py

```diff
@@ -15,6 +15,26 @@ def can_enroll_in_course(self, course):
         )
 
 
+def bio_image_upload_path(instance, _filename):
+    # Get the appropriate semester abbreviation
+    today = timezone.now().date()
+    semester = "fa" if today.month > 6 else "sp"
+    return semester + today.strftime("%y") + "/" + str(instance.user.pk)
+
+
+class MentorBioInfo(models.Model):
+    """
+    Includes biographical information about a mentor. Only one should exist per mentor;
+    edits to the bio should edit the existing instance in the database.
+    """
+    user = models.OneToOneField(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    first_name = models.CharField(max_length=100, verbose_name="First Name")
```

Comment:
Why aren't we just using the `first_name` and `last_name` from the associated `user`?